docs/changelog/1631.doc.rst

```diff
@@ -1 +1 @@
-Fixed grammar in top-level documentation. :user:tfurf
+Fixed grammar in top-level documentation - by :user:`tfurf`
```

Comment:
Hey @tfurf, here's the correct syntax for RST roles FYI ^

Reply:
Thanks! A pre-commit hook would be nice addition here.

Reply:
Yeah... Not sure if it should be in pre-commit or in the bot tho.